setup.py

```diff
@@ -13,7 +13,7 @@
     author='Vasu Kulkarni',
     author_email='vasu@redhat.com',
     install_requires=[
-        'apache-libcloud',
+        'apache-libcloud==3.3.0',
```

Comment:
I'm thinking we should avoid pinning to an exact version, and we should require this as a minimal version instead (`libcloud>=3.3.0`). It will be easy to forget about this pinned version number and then we'll be stuck on an old version.

Reply:
Exact pinning allows us to be reliable by preventing a fatal change breaking the existing implementation. I agree that it wouldn't break in new changes. Other than this, I don't see a reason why we shouldn't pick the latest versions. Let me go ahead and change to `>=`